code/modules/antagonists/villain/neu_vampires/covens/coven_powers/potence.dm

```diff
@@ -31,10 +31,12 @@
 /datum/coven_power/potence/one/activate()
```

Comment:
this is too cumbersome . just give them temporal STR

Reply:
that is the easy path, first of all i can't give HALF STR, the only thing i could give is 1 STR so it would be one STR per each level of potence and i think that would be too strong.